prov/efa/src/rdm/efa_rdm_ope.c

```diff
@@ -738,8 +738,6 @@ void efa_rdm_rxe_report_completion(struct efa_rdm_ope *rxe)
 					       rxe->total_len -
 					       rxe->cq_entry.len);
 
-		rxr_rm_rx_cq_check(ep, rx_cq);
```

Comment:
Did this actually cause a hang in application?

Reply:
I did not see it cause a hang actually, potentially because application polls tx cq efficiently so that it is never full.